packages/adders/vitest.config.ts

```diff
@@ -7,10 +6,9 @@ const ONE_MINUTE = 1000 * 60;
 export default defineConfig({
 	test: {
 		include: ['_tests/**/test.{js,ts}'],
-		testTimeout: ONE_MINUTE * 2,
+		testTimeout: ONE_MINUTE * 3,
 		hookTimeout: ONE_MINUTE * 3,
-		maxConcurrency: cpus().length,
-		pool: 'forks',
+		maxConcurrency: 10,
```

Comment:
Is there a reason for the `maxConcurrency` simplification? Having only 6 cores and 12 threads this overloads my computer and results in many timeouts. Reducing it down to 6 solved most occurences of these issues